packages/oui-button/_mixins.less

```diff
@@ -184,24 +188,27 @@
   .button-icon-base(
     @icon-selector,
     @icon-size: @oui-button-icon-icon-size,
+    @icon-size_large: @oui-button-icon-icon-size_large,
     @icon-color: @oui-button-icon-icon-color,
-    @icon-color_disabled: @oui-button-icon-right-icon-color_disabled
+    @icon-color_disabled: @oui-button-icon-icon-color_disabled
   ) {
-
     .@{icon-selector} {
-      vertical-align: text-bottom;
```

Comment:
Watch out: I think this property was there because on IE10 and more there was some unexpected behavior, be sure that QA test it on all browser and nothing is broken.

Reply:
Oh, ok. If necessary, we'll put it back.